consensus/config.go

```diff
@@ -2,15 +2,18 @@ package consensus
 
 import (
 	"time"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
 )
 
 type ParticipantConfig struct {
-	TimeoutInitial             time.Duration // the initial timeout for the pacemaker
-	TimeoutMinimum             time.Duration // the minimum timeout for the pacemaker
-	TimeoutAggregationFraction float64       // the percentage part of the timeout period reserved for vote aggregation
-	TimeoutIncreaseFactor      float64       // the factor at which the timeout grows when timeouts occur
-	TimeoutDecreaseFactor      float64       // the factor at which the timeout grows when timeouts occur
-	BlockRateDelay             time.Duration // a delay to broadcast block proposal in order to control the block production rate
+	TimeoutInitial             time.Duration           // the initial timeout for the pacemaker
+	TimeoutMinimum             time.Duration           // the minimum timeout for the pacemaker
+	TimeoutAggregationFraction float64                 // the percentage part of the timeout period reserved for vote aggregation
+	TimeoutIncreaseFactor      float64                 // the factor at which the timeout grows when timeouts occur
+	TimeoutDecreaseFactor      float64                 // the factor at which the timeout grows when timeouts occur
+	BlockRateDelay             time.Duration           // a delay to broadcast block proposal in order to control the block production rate
+	BlockTimestamp             hotstuff.BlockTimestamp // validator of block timestamps
```

Comment:
same renaming suggestion: `BlockTimestamp` -> `Blocktimer`
```suggestion
	Blocktimer             hotstuff.Blocktimer // validator of block timestamps
```